javascript/javascript1/week1/week1.js

```diff
@@ -0,0 +1,44 @@
+//Age-ify (A future age calculator)
+const yearOfBirth = 1984;
+const yearFuture = 2027;
+
+const age = yearFuture - yearOfBirth;
+
+console.log('You will be ', age, ' years old in', yearFuture);
```

Comment:
see diff .. remember to required space
console.log('You will be ', age, ' years old in ', yearFuture);

Reply:
Hi Rajpal, 

Do you mean that there should be no space before age and yearFuture